### PR TITLE
Update gssp config without install

### PR DESCRIPTION
**Pull request by phavekes**
Add a new option to the deploy-scipt ```-c|--config-only```. This wil cause only the components configuration to be updated, without any intrusive actions (like extracting tar's, copying default files, creating/updating users). This way an updated configuration (e.g. new AzureMFA institutions.yml, or WebAuthn trusted certificates) van be send to app-servers, without interrupting users.

### CODE REVIEW

File: roles/deploy/tasks/activate.yml [hunk 1]
```diff
@@ -17,3 +17,4 @@
 
 - name: Activate component
   file: src={{ component_dir_name }} dest=/opt/www/{{ component_vhost_name }} state=link
+  when: not component_info.failed
```
Comment (pmeulen) [inline roles/deploy/tasks/activate.yml:20]: You meant `when: not configonly | bool` here, right?

File: roles/deploy/tasks/main.yml [hunk 2]
```diff
@@ -241,25 +245,25 @@
 
   # The dev role installed nvm (node version manager) for nodejs
   # Read the NODE_VERSION form component_info (if any)
-  - name: Fetch nodejs version from component_info
+  - name: DEVELOP - Fetch nodejs version from component_info
```
Comment (pmeulen) [inline roles/deploy/tasks/main.yml:248]: Nitpick: good fix, adding DEVELOP here and fixing typo's. But should not be part of this PR. It makes it less obvious what was changed.

File: scripts/deploy.sh [hunk 3]
```diff
@@ -48,6 +49,7 @@ if [ -z "${COMPONENT_TARBALL}"  ]; then
     echo "-l|--limit: <SUBSET>   Limit option to pass to ansible (limits hosts)"
     echo "-K|--ask-sudo-pass     Ask for sudo password"
     echo "-n|--no-unarchive      Skip uploading and unarchiving the tarball on the remote"
+    echo "-c|--config-only       Only update the components configuration files (only for AzureMFA-gssp and WebAuthn-gssp)"
```
Comment (pmeulen) [inline scripts/deploy.sh:52]: Suggestion: add a list of components that support the -c option and give an error when the -c option is used with a component that does not implement it?

**Review by pmeulen — CHANGES_REQUESTED** [review]
Need to fix the wrong condition.
Rest looks good.
Let's discuss whether giving an error when a component does not support -c is worthwhile.

File: scripts/deploy.sh (hunk 3 above)
Comment (phavekes) [reply scripts/deploy.sh:52]: Nice idea, I'll try to implement this.

**Review by pmeulen — APPROVED** [review]

**Comment by pmeulen** [comment]
Looks good!
